Test mode UI changes

sameer79 opened the pull request:
This PR for multiple issues because all are inter-related.
Issue addressed in this PR are below 
1] ISSUE-899
2] ISSUE-902
3] ISSUE-901

arunmahadevan commented:
This seem to include the changes from @HeartSaVioR ? If so lets wait until those PRs are merged.

sameer79 commented:
@arunmahadevan  sorry my bad. I missed reverting the backend PR's.
Reverted the commits and updated the PR with proper UI code.
You can merge the backend PR and this one and test out everything.

shahsank3t commented on webservice/src/main/resources/app/scripts/components/TopologyGraphComponent.jsx:
```diff
@@ -768,7 +768,7 @@ export default class TopologyGraphComponent extends Component {
           nestedFields(fieldObj[key], levelTemp);
         } else {
-          list.push(`<li style='padding-left : ${level > 0 ? (10*level) : level}px' class=${level > 0 ? 'demo': ''}> ${fieldObj[key]} <span class='output-type'> ${key}</span> </li>`);
+          list.push(`<li style='padding-left : ${level > 0 ? (10*level) : level}px' class=${level > 0 ? 'demo': ''}> ${key} <span class='output-type'> ${fieldObj[key]}</span> </li>`);
```
We want to keep the styling of the value and field name as it currently is, just replace the position.

shahsank3t commented on webservice/src/main/resources/app/scripts/containers/Streams/TestRunComponents/EventLogContainer.jsx:
```diff
@@ -34,18 +34,18 @@ class EventLogContainer extends Component{
-        body.appendChild(eventOverlay);
-      }
-    }
+    // if(!this.completed){
```
We can get rid of this commented code, no need of it now.

shahsank3t commented on webservice/src/main/resources/app/scripts/containers/Streams/TopologyEditor/TopologyEditorContainer.jsx:
```diff
@@ -1231,41 +1233,51 @@ class TopologyEditorContainer extends Component {
+                          this.setState({testRunningMode : false}, () => {
+                            clearInterval(this.interval);
+                            const msg =  this.state.abortTestCase
+                                          ? "Test Run has been Abort successfully"
```
Change the msg to "Test Run aborted successfully"

shahsank3t commented:
@sameer79 Went through the code and left few comments that you might want to address.

sameer79 commented:
@shahsank3t   Address all review comments.

sameer79 commented:
@arunmahadevan  @shahsank3t  Fixed the other UI bugs and integrated killing of test case into UI
